internal/provider/resource_access_generic.go

```diff
@@ -25,6 +25,7 @@ import (
 	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
 	"github.com/hashicorp/terraform-plugin-log/tflog"
 	"github.com/juju/names/v5"
+	namesv4 "github.com/juju/names/v5"
```

Comment:
Ah good catch. It's indeed not needed since we create the offer tag and the validation only takes place in ParseTag(). Removed all the uses of names/v4